src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -3,51 +3,63 @@ title: Alerts
 sidebar_order: 1
 ---
 
-Sentry users with admin permissions or higher can create one of two types of alerts: [Metric Alerts](#metric-alerts) or [Issue Alerts](#issue-alerts). You can use [Metric Alerts](#metric-alerts) for high-level monitoring of patterns, or fine-grained monitoring of individual events. An [Issue Alert](#issue-alerts) fires whenever any issue in the project matches specified criteria, such as a resolved issue re-appearing or an issue affecting many users. To confirm or set admin permissions, see **Project Settings > Alerts > New Alert**. 
+Sentry users with admin permissions or higher can create alerts. To confirm or set admin permissions, see **Project Settings > Alerts > New Alert**.
 
-## Metric Alerts
+## For Errors
 
-{% capture __alert_content -%}
-Metric Alerts are currently in beta and available for [Team and Business plans](https://sentry.io/pricing/). For more information about access to Metric Alerts, feel free to reach out at [alerting-feedback@sentry.io](mailto:alerting-feedback@sentry.io).
-{%- endcapture -%}
-{%- include components/alert.html
-    title="Note"
-    content=__alert_content
-    level="warning"
-%}
+Sentry automatically groups individual error events into Issues based on stack trace and other factors. For more details about grouping, see the [full documentation on Grouping & Fingerprints]({%- link _documentation/data-management/event-grouping/index.md -%}).
+
+Sentry has two types of alerts you can use for errors:
+
+[Issue Alerts](#issue-alerts)
 
-Metric alerts allow you to filter and set thresholds on all errors across a project. Use metric alerts for high-level monitoring of patterns or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors or volume of database errors.
+: Notifies you when individual Sentry [Issues]({%- link _documentation/data-management/event-grouping/index.md -%}) match your alerting criteria, such as an Issue happening more frequently than usual.
+
+[Metric Alerts](#metric-alerts)
+
+: A metric is the value of an aggregate function like `count()` or `avg()` applied to your event data. Metric alerts can be used on error data to compute the number of events or the number of users affected by events in your project over a period of time. You can filter events by attributes and tags, which can be particularly useful when a set of events is not grouped into a single issue, making issue alerts unsuitable to use.
```

Comment:
perhaps flip sentence 1 and 2? So, define what a metric alert is, then define what a metric can be?

Also, the final sentence needs a bit more clarity - perhaps "You can filter events by attributes and tags, which is particularly useful to aggregate across events that are not grouped into a single Issue. When this happens, metric alerts are the most useful form of alert." or something similar?